### ISSUE TITLE
Broken project launch and hot reload.

### ISSUE THREAD

**Issue by xxyuk** [arackaf/customize-cra#204]
Hello, i have a problem when i use this plugin.
The project create by create-react-app, i use 'yarn start',it works.
After I used customize-cra to add a simple webpack plugin, I started the project again and the project went blank. If I wanted to see the content, I had to force a refresh of the browser to succeed. I have tried many times, and when I closed customize-cra, the project can be used normally, I want to know if you have encountered this problem.

Looking forward to your reply, thank you very much!

<img src="https://user-images.githubusercontent.com/41352992/71455018-2d997780-27ce-11ea-9cd2-da63a9038566.png" />

**Comment by tridungle** [comment]
> Hello, i have a problem when i use this plugin.
> The project create by create-react-app, i use 'yarn start',it works.
> After I used customize-cra to add a simple webpack plugin, I started the project again and the project went blank. If I wanted to see the content, I had to force a refresh of the browser to succeed. I have tried many times, and when I closed customize-cra, the project can be used normally, I want to know if you have encountered this problem.
> 
> Looking forward to your reply, thank you very much!
> 
> ![](https://user-images.githubusercontent.com/41352992/71455018-2d997780-27ce-11ea-9cd2-da63a9038566.png)

I have the same problem. I think there are something wrong with addWebpackPlugin. It work fine when I remove addWebpackPlugin from the config-overrides.js

**Comment by jbberinger** [comment]
Ran into the exact same issue. Had to disable the plugin entirely to avoid the refresh problem.

**Comment by zsirfs** [comment]
@jbberinger When I remove the plugin, it returned to normal. Why is it?

**Comment by jbberinger** [comment]
@zsirfs I'm not sure.

**Comment by XianZhengquan** [comment]
不是 `customize-cra` 引起的问题，我将 CRA的项目eject之后，在webpack.config.js中，添加这个 `antd-dayjs-webpack-plugin` plugin，启动项目依然是白屏，需要刷新
